src/map/script.c

```diff
@@ -10574,6 +10580,12 @@ static BUILDIN(guildopenstorage)
 	if (sd == NULL)
 		return true;
 
+	// Mapflag preventing from openstorage here
+	if (!pc_has_permission(sd, PC_PERM_BYPASS_NOSTORAGE) && map->list[sd->bl.m].flag.nogstorage&2) {
+		script_pushint(st, 0);
```

Comment:
For guildopenstorage, 0 means success and non zero means failure, probably change to 1(existing error code) or 3(new error code)